### PR TITLE
Indicating abbr.

### PR DESCRIPTION
**Pull request by xinbenlv**
Indicating Application Binary Interface is abbreviated as "ABI"

### CODE REVIEW

File: docs/abi-spec.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Application Binary Interface Specification
 Basic Design
 ============
 
-The Application Binary Interface is the standard way to interact with contracts in the Ethereum ecosystem, both
+The Application Binary Interface(ABI) is the standard way to interact with contracts in the Ethereum ecosystem, both
```
Comment (chriseth) [inline docs/abi-spec.rst]: Please add a space in front of `(`

Comment (xinbenlv) [reply docs/abi-spec.rst]: Done

**Comment by chriseth** [comment]
Sorry, but could you please squash this into a single commit?

**Review by axic — APPROVED** [review]

**Comment by axic** [comment]
Docs passed, merging.

**Pull request merged**